db/migrate/20190412181103_remove_availability_from_volunteers.rb

```diff
@@ -1,5 +1,14 @@
 class RemoveAvailabilityFromVolunteers < ActiveRecord::Migration[5.2]
   def down
```

Comment:
Tilneil, this isn't quite correct.  The down, up business is kind of obsolete, and if used, down should do the opposite of up.   (The down part is invoked during rollback.) I think you want just one part, that says def change remove_column :volunteers, :availability, :string if availability_exists?  Also, I think you should create a new migration file for this with a later timestamp than any that you have, and delete this file -- as it has an earlier timestamp, it might not run, when people have run other migration files with a later timestamp.